### ISSUE TITLE
JAWS on Edge reads out entire href of error summary link

### ISSUE THREAD

**Issue by owenatgov** [alphagov/govuk-frontend#4790]
## Description of the issue

When using JAWS on Edge, the error summary reads out the entire href of the page it's on as well as the error itself, making the announcement needlessly verbose. This was first noticed during the testing for https://github.com/alphagov/govuk-frontend/pull/2677 and was later reported by a user in [this zendesk ticket
](https://govuk.zendesk.com/agent/tickets/5636807).

## Steps to reproduce the issue

1. In Edge using JAWS, go to https://govuk-frontend-review.herokuapp.com/examples/error-summary-with-messages
2. Navigate to the first error summary link

### Expected

JAWS only announces the error's text, announcing that it's a link.

### Actual

JAWS announces the error summary text and the entire href of the page (likely the link itself as error summary links are anchor links)

## Environment (where applicable)

- Browser: Edge
- Browser version: 121
- JAWS version: 2022.2207.25
- GOV.UK Frontend Version: 5.1.0

**Comment by querkmachine** [comment]
Retested I've retested this in Edge 144 and multiple versions of JAWS and couldn't reproduce it. Instead, the text of the link is announced followed by 'same page link'. This is consistent for other same page links like the skip link and is not unique to the Error summary component.

Tested in JAWS versions:
- 2022.2207.25
- 2023.2303.114
- 2024.2411.16
- 2025.2412.50 

As I cannot reproduce it in the same version of JAWS and we haven't made any changes to the error summary links, I can only surmise that this was an issue with Edge that has since been resolved. If it recurs again, we can reopen this issue.
